src/PIL/Image.py

```diff
@@ -1764,7 +1764,7 @@ def remap_palette(self, dest_map, source_palette=None):
 
         return m_im
 
-    def resize(self, size, resample=NEAREST, box=None):
+    def resize(self, size, resample=NEAREST, box=None, max_reduce=None):
```

Comment:
Maybe the better name is `reduce_gap` since this is actually minimal distance (in terms of scale) between the `reduce()` method result size and the requested size.

Reply:
Renamed to `reducing_gap`